core/src/com/ericc/the/game/Enums.java

```diff
@@ -0,0 +1,25 @@
+package com.ericc.the.game;
+
+public class Enums {
+    public enum TILETEXTURE {
+        WALL_LEFT,
```

Comment:
ja bym to przerobił na dwa enumy: jeden określający teksturę (w sensie czy to jest generalnie ściana murowana czy podłoga z błota czy trawa) i drugi określający jaki konkretnie kafelek to jest z tileseta, czyli te bity sąsiedztwa które teraz trzymamy jako string. Bity sąsiedztwa chyba wgl najlepiej trzymać jako jeden bajt i korzystać z operacji bitowych zamiast porównywania stringów.